client/blocks/poll/sidebar.js

```diff
@@ -146,6 +146,11 @@ const SideBar = ( {
 
 	const answerStyle = getAnswerStyle( attributes, className );
 
+	// Force redirect into simple THANK_YOU until some workaround to prevent phishing
```

Comment:
Can we maybe open a GitHub issue with a slightly better explanation for this and just link to it here instead?
Will probably make things easier for when we actually get around to this.

Reply:
I've created #12 with a succinct issue description, getting the link on the commented code now. We can further edit the issue if we feel like explaining more of it.